### PR TITLE
add dialog to export and import settings text files

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- fixes a bug where settings are not displayed correctly in ui after  #"import"
- `copyFrom/ToClipboard` has stupid limitations in MP, so let the user press Ctrl+C/V on a edit box instead

It scrolls for more than 40 settings and can display thousands.

### CODE REVIEW

**Comment by Dorbedo** [comment]
I think, it would be an advantage, if the settings would be exported sorted by category. Maybe even with a comment.
``` 
// category 1
setting_category1_A = 1;
force setting_category1_B = 2;
// category 2
setting_category2_A = ABC;
```
I recognized this feature as missing while exporting the ace-settings

**Comment by Dystopian** [comment]
it would be cool if only changed settings could be exported

**Comment by commy2** [comment]
^ fixes https://www.youtube.com/watch?v=F-whgpGviRU&feature=youtu.be

**Comment by commy2** [comment]
^ sorted by category

**Comment by commy2** [comment]
Done and done.

File: addons/settings/fnc_export.sqf [hunk 1]
```diff
@@ -15,9 +15,10 @@ Author:
 ---------------------------------------------------------------------------- */
 #include "script_component.hpp"
 
-params [["_source", "client", [""]]];
+params [["_source", "client", [""]], ["_skipDefault", false, [false]]];
```
Comment (Dystopian) [inline addons/settings/fnc_export.sqf]: add `_skipDefault` to header?

**Comment by Dystopian** [comment]
Tested, works great.
If have few non-default settings and uncheck `Show default values`, comments are really superfluous. It would be good to have checkbox to disable comments (and maybe double newline between categories).

**Comment by commy2** [comment]
I think the comments are great and a checkbox just to disable them feels like "too much".

**Pull request merged**